packages/xstate-react/src/useService.ts

```diff
@@ -1,39 +1,30 @@
 import { useMemo } from 'react';
-import { EventObject, State, Interpreter, Typestate } from 'xstate';
-import { useSubscription, Subscription } from 'use-subscription';
+import { EventObject, State, Interpreter, Typestate, Sender } from 'xstate';
+import { useActor } from './useActor';
+import { ActorRef } from './types';
+
+export function fromService<TContext, TEvent extends EventObject>(
+  service: Interpreter<TContext, any, TEvent, any>
+): ActorRef<TEvent, State<TContext, TEvent>> {
+  const { machine } = service;
+  return {
+    send: service.send.bind(service),
+    subscribe: service.subscribe.bind(service),
+    stop: service.stop,
```

Comment:
general q: should this always be exposed? should any owner of an actor ref be allowed to stop a particular actor? or should this be restricted only to the immediate parent?

Reply:
It should in theory be "restricted" but it should be exposed to `useService`.

Reply:
Good question - currently it's part of the interface, but yes, an actor can stop at any time for any reason. Let's leave this in here for now, since we don't expose `.stop()` to the `useActor` consumer.